### DISCUSSION TITLE
Problem with Rust and Lazyvim

### DISCUSSION THREAD

**Discussion by bells307** [neovim/neovim#29471] [Q&A]
Hello everyone, please tell me if anyone has encountered this problem.

I use lazyvim, in its extras I included plugins for rust. But when I create a new module in text, then using Code action I generate the module’s rs file, and when I try to go using Go to definition (gd), the following error appears. Moreover, the module file itself is created, but it is not possible to navigate through gd.

![telegram-cloud-photo-size-2-5463176183102429808-y](https://github.com/neovim/neovim/assets/45540153/f9bbc9ca-c225-4818-b74b-a19ac331381b)

I work on macos, but on the neighboring arch linux with the same config it runs fine. Has anyone encountered this? I'll attach a gif to show you

![nvim](https://github.com/neovim/neovim/assets/45540153/988a4285-ad9c-453b-9bc4-fa95af2858b5)

There is my neovim config:

https://github.com/bells307/dotfiles/tree/master/.config/nvim

(sorry for my english)

**Comment by clason** [comment]
Please use the LazyVim support channels then.
